src/amo/components/Categories.js

```diff
@@ -46,14 +60,27 @@ export class CategoriesBase extends React.Component {
 
     return (
       <div className="Categories">
-        <ul className="Categories-list"
-          ref={(ref) => { this.categories = ref; }}>
-          {Object.values(categories).map((category) => (
-            <li key={category.slug} className="Categories-list-item">
-              <Link className="Categories-link"
-                to={`/${visibleAddonType(addonType)}/${category.slug}/`}>
-                {category.name}
-              </Link>
+        {loading ? (
+          <div className="Categories-loadingText visually-hidden">
+            {i18n.gettext('Loading categories.')}
+          </div>
+        ) : null}
+        <ul
+          className="Categories-list"
+          ref={(ref) => { this.categories = ref; }}
+        >
+          {categories.map((category) => (
+            <li className="Categories-list-item" key={category.slug}>
```

Comment:
Probably not a huge deal but that's a good point. I'll grab the index and put that in there if `category.slug` is undefined.